packages/core/src/ceramic.ts

```diff
@@ -506,6 +506,7 @@ class Ceramic implements CeramicApi {
    * @param doctype - Document type
    * @param params - Create parameters
    * @param opts - Initialization options
+   * @deprecated
    */
   async createDocument<T extends Doctype>(doctype: string, params: DocParams, opts?: DocOpts): Promise<T> {
```

Comment:
Mind also adding a `console.warn` here telling devs that it's deprecated?

Reply:
I hope to delete this method entirely ASAP and before the next release.  The only reason it isn't done in this PR is because of this compiler error I mentioned: https://github.com/ceramicnetwork/js-ceramic/pull/1180#discussion_r604348910